test/models/work_test.rb

```diff
@@ -0,0 +1,100 @@
+require "test_helper"
+
+describe Work do
+  describe 'validations' do
+    let(:work) { Work.new(category:"movie", title: "Pee-Wee's Big Adventure") }
+
+    it "must be valid" do
+      value(work).must_be :valid?
+    end
+
+    it "must be invalid without a category" do
+      work.category = nil
+      value(work).wont_be :valid?
+    end
+
+    it "must be invalid without a title" do
+      work.title = nil
+      value(work).wont_be :valid?
+    end
+
+    it "must be valid with a supported category type" do
+      categories = ["movie", "book", "album"]
+      categories.each do |cat|
+        work.category = cat
+        value(work).must_be :valid?
+      end
+    end
+
+    it "must be invalid without a supported category type" do
+      work.category = "literally a bunch of junk"
+      value(work).wont_be :valid?
+    end
+  end
+
+  describe 'self methods' do
+    it 'returns an array of works by category sorted by vote count in reverse' do
```

Comment:
Good test. I'd love to see you think about how to DRY up this test and write the logic one time and then be able to iterate through each category type.